controllers/provisioners/eks/eks.go

```diff
@@ -26,7 +27,8 @@ import (
 )
 
 const (
-	ProvisionerName = "eks"
+	ProvisionerName                     = "eks"
+	defaultLaunchConfigurationRetention = 2
```

Comment:
In a separate PR, we might want to make it configurable (or at least have some annotation that can override this).

Reply:
Yes, I was trying to come up with a way to not add this to the CR, but using a controller flag also doesnt make much sense, because not all provisioners care about launch configurations (e.g. eks-managed).

Reply:
Annotation is a good idea!